include/xtensor/xfft.hpp

```diff
@@ -0,0 +1,294 @@
+#ifndef XFFT_HPP
+#define XFFT_HPP
+
+#include "xmath.hpp"
+#include "xcomplex.hpp"
+#include "xutils.hpp"
+#include "xaxis_slice_iterator.hpp"
+#include "xio.hpp"
+#include "xview.hpp"
+
+namespace xt 
+{
+    namespace fft
+    {
+        template<typename T = double, typename E1, typename E2>
+        xt::xarray<std::complex<T>> fftconvolve(E1&& xvec, E2&& yvec, std::ptrdiff_t axis = -1);
+        
+        namespace detail
+        {
+            template<typename T>
+            T PI()
+            {
+                constexpr double PI = 3.141592653589793238463;
+                return PI;
+            }
+
+            size_t reverseBits(size_t x, int n)
```

Comment:
I'm pretty sure that lower-case only is used everywhere (applies at several places)

Reply:
Done